docs/contentEditableDisabled/index.html

```diff
@@ -7,84 +7,164 @@
+                <li><code>formatjustifycenter</code></li>
+                <li><code>formatjustifyfull</code></li>
+                <li><code>formatjustifyleft</code></li>
+                <li><code>formatjustifyright</code></li>
+                <li><code>formatbackcolor</code></li>
+                <li><code>formatbold</code></li>
+                <li><code>insertlink</code></li>
+                <li><code>formatfontname</code></li>
+                <li><code>formatfontsize</code></li>
+                <li><code>formatfontcolor</code></li>
+                <li><code>formatindent</code></li>
+                <li><code>inserthorizontalrule</code></li>
+                <li><code>insertorderedlist</code></li>
+                <li><code>insertunorderedlist</code></li>
+                <li><code>formatitalic</code></li>
+                <li><code>formatoutdent</code></li>
+                <li><code>formatremove</code></li>
+                <li><code>formatstrikethrough</code></li>
+                <li><code>formatsubscript</code></li>
+                <li><code>formatsuperscript</code></li>
+                <li><code>formatunderline</code></li>
+            </ul>
+        </p>
+        <p class="issue" data-number="249"></p>
+        <h3>
+            <code>contenteditabledisabled</code> Allowed Token Shortcuts.
+        </h3>
+        <p>
+            The following tokens are also supported by the <code>contenteditabledisabled</code> attribute and serve as shorthands for specifying a set of tokens from contenteditabledisabled Allowed Tokens:
+        </p>
+        <p>
+            <ul>
+                <li><code>formatjustify</code>: combines <code>formatjustifycenter</code>, <code>formatjustifyfull</code>, <code>formatjustifyleft</code>, and <code>formatjustifyright</code>.</li>
+                <li><code>formatfont</code>: combines <code>formatfontname</code>, <code>formatfontsize</code>, and <code>formatfontcolor</code></li>
+                <li><code>formatdent</code>: combines <code>formatindent</code> and <code>formatoutdent</code></li>
+                <li><code>insertlist</code>: combines <code>insertorderedlist</code> and <code>insertunorderedlist</code></li>
+                <li><code>formatscript</code>: combines <code>formatsubscript</code> and <code>formatsuperscript</code></li>
+            </ul>
+        </p>
+        <p class="note">
+            Note that this specification does not have a shortcut to disable all editing UI.  Authors should consider using a <code>textarea</code> for this purpose.
+        </p>
+    </section>
+    <section id="element-contenteditable-mixin">
         <h2>Extensions to the <dfn>ElementContentEditable</dfn> mixin</h2>
         <pre class="idl">
-           partial interface ElementContentEditable {
-             [SameObject, PutForwards=value] readonly attribute DOMCommandTokenList contentEditableDisabled;
-           };
+            [Exposed=Window]
+            interface ElementContentEditable {
```

Comment:
Ok, this can't go here unfortunately, because it would break HTML. 
https://html.spec.whatwg.org/#contenteditable

This needs to be a PR on HTML itself as part of  the `ElementContentEditable` mixin defined there. 

The values probably need to go into HTML also, but they can point back to our spec, which would defined the expected behavior for each token.

Reply:
Wouldn't `partial interface mixin` be enough here?

Reply:
@marcoscaceres if we move this into HTML spec then I guess, all of it can just live there.
contenteditabledisabled could just be another attribute on conenteditable and we won't need another spec.
@annevk as FYI

Per previous discussion, I created an issue on HTML spec asking for feedback on this: https://github.com/whatwg/html/issues/5863